packages/sanity/src/desk/structureBuilder/Intent.ts

```diff
@@ -2,45 +2,61 @@ import {PartialDocumentList, getTypeNamesFromFilter} from './DocumentList'
 import {StructureNode} from './StructureNodes'
 
 /**
- * @hidden
- * @beta */
+ * Type for intent parameters (json)
+ *
+ * @public
+ */
 export type IntentJsonParams = {[key: string]: any}
 
 /**
- * @hidden
- * @beta */
+ * Type for base intent parameters
+ *
+ * @public */
 export type BaseIntentParams = {
```

Comment:
could be converted to interface

```suggestion
export interface BaseIntentParams {
```